profile/README.md

```diff
@@ -0,0 +1,41 @@
+![Foreman Header](https://raw.githubusercontent.com/theforeman/.github/main/profile/foreman-header.png)
```

Comment:
This is a 404. I wonder if this can't just be a relative path.

Reply:
I can see it now as it is merged. I took this from other projects and planned it with this in mind. I had not found one with a relative path, only this or an external URL.

Reply:
#3